### DISCUSSION TITLE
Immich in Proxmox container (Docker), photo in sharing TrueNas.

### DISCUSSION THREAD

**Discussion by Master-of-Mouse** [immich-app/immich#9078] [Q&A]
Good morning, I created a Proxmox container and installed Docker.
In this CT I installed Immich following the guide.
I would like to keep Immich and its database in the container but I would like all the photos/videos to be in a TrueNas share that I have in another VM.

This is because the VMs and CTs are on an SSD, but the storage is on HDD managed by TrueNas.

Has anyone already done a similar setup?
How could this be done then?

Perhaps by creating a share in TrueNas, mounting it from the shell in the CT and indicating the path in the .env file under UPLOAD_LOCATION=./library ?

**Comment by alextran1502** [comment]
I have a similar setup. I mounted TrueNAS as NFS to Promox and then attached the storage to the VM and used it in the `UPLOAD_LOCATION`

**Comment by Master-of-Mouse** [comment]
I found out that in proxmox, the non-privileged containers, don't have access to SMB/CIFS and NFS so it skips everything.
I will prepare a VM with Docker and use it for the various CTs.

**Comment by silbaer** [comment]
You can mount the NFS/SMB on the proxmox host and use bind mounts to mount this in your LCX container: https://itsembedded.com/sysadmin/proxmox_bind_unprivileged_lxc/
